thermo_nerf/evaluator/evaluator.py

```diff
@@ -77,7 +77,7 @@ def _compute_metrics(
             ) = self._pipeline.model.get_image_metrics_and_images(outputs, batch)
 
             # Save imgs
-            images_dict["rgb"] = images_dict.pop("img")
+            images_dict[RenderedImageModality.RGB.value] = images_dict.pop("img")
```

Comment:
Here is the code to change the naming convention for `image_dict["img"]` in nerfstudio to our own modality types. This is actually for thermonerf and nerfacto model as they directly use the nerfacto model in nerfstudio. 

For concat-nerf, if without this code in the evaluator, we could directly use `image_dict[modality.RGB.value]`. With this code, we should use  `image_dict["img"]` in the conat-nerfacto-model.

Reply:
Then we should switch to "img" instead of "rgb" so we avoid those pop operation which are a little unclear and instead we put your "special case" just for rgb in the output of concat-nerf? Does that sounds good. This way it's closer to nerfstudio but also it's less places with switch between enum values